docs/questions/query-builder/expressions/sumif.md

```diff
@@ -0,0 +1,248 @@
+---
+title: SumIf
+---
+
+# SumIf
+
+`SumIf` adds the values in a column based on the values in another column.
+
+Syntax: `SumIf(column, condition)`.
+
+Example: in the table below, `SumIf([Payment], [Plan] = "Basic")` would return 200.
+
+| Payment  | Plan        |
+|----------|-------------|
+| 100      | Basic       |
+| 100      | Basic       |
+| 200      | Business    |
+| 200      | Business    |
+| 400      | Premium     |
+
+> [Aggregation formulas](../expressions-list.md#aggregations) like `sumif` should be added to the query builder's [**Summarize** menu](../../query-builder/introduction.md#summarizing-and-grouping-by) > **Custom expression** (scroll down in the menu if needed).
+
+## Parameters
+
+- `column` can be the name of a numeric column, or an expression that returns a numeric column.
+- `condition` is an expression that returns `true` or `false`.
+
+## Multiple conditions
+
+| Payment  | Plan        | Date Received     |
+|----------|-------------| ------------------|
+| 100      | Basic       | October 1, 2020   |
+| 100      | Basic       | October 1, 2020   | 
+| 200      | Business    | October 1, 2020   |
+| 200      | Business    | November 1, 2020  |
+| 400      | Premium     | November 1, 2020  |
+
+To sum a column based on multiple _mandatory_ conditions:
+
+```
+SumIf([Payment], [Plan] = "Basic" AND month([Date Received]) = 10)
+```
+
+This expression would return 200 on the sample data above.
+
+To sum a column with multiple _optional_ conditions:
+
+```
+SumIf([Payment], ([Plan] = "Basic" OR [Plan] = "Business"))
+```
+
+Returns 600 on the sample data.
+
+To combine mandatory and optional conditions:
+
+```
+SumIf([Payment], ([Plan] = "Basic" OR [Plan] = "Business") AND month([Date Received]) = 10)
+```
+
+Returns 400 on the sample data.
+
+> Tip: make it a habit to put brackets around your `AND` and `OR` statements to avoid making mandatory conditions optional (or vice versa).
+
+## Conditional subtotals by group
+
+To get a conditional subtotal for a category or group, you'll:
+
+1. Write a `sumif` formula with your conditions.
+2. Add a [**Group by** column](../../query-builder/introduction.md#summarizing-and-grouping-by) in the query builder.
+
+| Payment  | Plan        | Date Received     |
+|----------|-------------| ------------------|
+| 100      | Basic       | October 1, 2020   |
+| 100      | Basic       | October 1, 2020   | 
+| 200      | Business    | October 1, 2020   |
+| 200      | Business    | November 1, 2020  |
+| 400      | Premium     | November 1, 2020  |
+
+To sum payments for the Business and Premium plans:
+
+```
+SumIf([Payment], [Plan] = "Business" OR [Plan] = "Premium")
+```
+
+Or, sum payments for all plans that aren't "Basic": 
+
+```
+{% raw %}SumIf([Payment], [Plan] != "Basic"){% endraw %} 
```

Comment:
`!=` gets rendered as "≠" on the site (even with the raw tags). When you copy the code block, you get the correct formula.

I reference the `!=` filter in the callout below. Any ideas to get the `!=` to show up here?

In this case, I'm avoiding writing "not equals filter" or "exclusion filter" since `<>` is invalid, and`NOT` might be possible too (though I haven't tried it).